### ISSUE TITLE
Broken link in Wiki...

### ISSUE THREAD

**Issue by jvautrot** [codeforboston/voiceapp311#348]
**Describe the bug**
In the Wiki section on "General Information About Alexa Development" there is a broken link.  The link to: [https://moduscreate.com/build-an-alexa-skill-with-python-and-aws-lambda](https://moduscreate.com/build-an-alexa-skill-with-python-and-aws-lambda) returns a 404 page.  The link should point to [https://moduscreate.com/blog/build-an-alexa-skill-with-python-and-aws-lambda](https://moduscreate.com/blog/build-an-alexa-skill-with-python-and-aws-lambda).  It looks like they moved their blog to a subdirectory.

**To Reproduce**
1. Go to https://github.com/codeforboston/voiceapp311/wiki/General-Information-About-Alexa-Development
2. Click on the link to the guide https://moduscreate.com/build-an-alexa-skill-with-python-and-aws-lambda
3. See the 404 page.

**Expected behavior**
The link should load the guide found here: https://moduscreate.com/blog/build-an-alexa-skill-with-python-and-aws-lambda

**So basically this:**
![Screenshot from 2020-12-17 16-46-46](https://user-images.githubusercontent.com/36384899/102554539-0a2ca400-4093-11eb-8beb-af781086f687.png)

**Should be this:**
![Screenshot from 2020-12-17 16-48-45](https://user-images.githubusercontent.com/36384899/102554570-157fcf80-4093-11eb-9747-4ff3468b09f7.png)

**By doing this:**

![311_Link_Diff](https://user-images.githubusercontent.com/36384899/102555469-d9e60500-4094-11eb-89f9-318949105231.png)
